src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map((movie) => <MovieCard key={movie.imdbId} movie={movie} />)}
```

Comment:
let's not write it in one line, it's good for codewars but outside codewars such code is really hard to read